### PR TITLE
Add support for DOS URLs

### PR DESCRIPTION
**Pull request by mikebaumann**
Added support for DOS URLs in files query and manifest export.
Include cloud native urls in manifest, at least for now, as a stop gap for export to FireCloud.

Primary reviewer: @caaespin

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ services:
       ES_PROTOCOL: "http"
       ES_FILE_INDEX: "fb_index"
       ES_DONOR_INDEX: "analysis_index"
+      DOS_DSS_SERVICE: "dos-dss.ucsc-cgp-dev.org"
```
Comment (coverbeck) [inline docker-compose.yml]: We will need to update dcc-ops with this setting. Nothing wrong with this, just FYI.

File: responseobjects/api_response.py [hunk 2]
```diff
@@ -234,6 +240,12 @@ def fetch_entry_value(mapping, entry, key):
         else:
             return None
 
+    @staticmethod
+    def compose_dos_url(file_uuid, file_version):
+        DOS_DSS_SERVICE = os.environ.get('DOS_DSS_SERVICE')
```
Comment (coverbeck) [inline responseobjects/api_response.py]: Should you have a default value? Or return nothing, if the environment variable is not set?

Maybe overkill...

File: docker-compose.yml (hunk 1 above)
Comment (coverbeck) [reply docker-compose.yml]: Created a PR for this: https://github.com/DataBiosphere/dcc-ops/pull/6

File: responseobjects/api_response.py (hunk 2 above)
Comment (caaespin) [reply responseobjects/api_response.py]: I agree that there should be a default value, whatever it's used for development. At least that's the pattern I've personally been using. It helps me stand up something quickly when I want to test something.

**Pull request merged**